_config.yml

```diff
@@ -130,6 +130,18 @@ defaults:
       path: _faq/security_implications
     values:
       category: Security implications
+  - scope:
+      path: _faq/resource
```

Comment:
I Think you can get away with one FAQ here.

`Google Cloud Platform Resource Coverage`

It's up to you though.

Reply:
Updated